### PR TITLE
CU-2pjkxye - No visible QR code

### PR DESCRIPTION
**Pull request by raulduartep**
**What current issue(s) from Trello/Github does this address?**

**What problem does this PR solve?**

**How did you solve this problem?**

**How did you make sure your solution works?**

**Are there any special changes in the code that we should be aware of?**

**Is there anything else we should know?**

- [ ] Unit tests written?

### CODE REVIEW

**Comment by lllwvlvwlll** [comment]
Task linked: [CU-2pjkxye *On next step to view the Encrypted QR code, resulted in no visible QR code to copy and no results when selecting copy and trying to paste.](https://app.clickup.com/t/2pjkxye)

File: app/hocs/withCopyCanvasToClipboard.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ export default function withCopyCanvasToClipboard(Component) {
       // $FlowFixMe
       const url = ref.toDataURL('image/jpg', 0.8)
       const base64Data = url.replace(/^data:image\/png;base64,/, '')
-      const storagePath = storage.getDefaultDataPath()
+      const storagePath = storage.getDataPath()
```
Comment (comountainclimber) [inline app/hocs/withCopyCanvasToClipboard.js]: I am pretty sure this is not going to give you the path we use everywhere else in the application... We should be using `const path = await ipcRenderer.invoke('getStoragePath')` here

File: app/containers/Encrypt/Encrypt.jsx [hunk 2]
```diff
@@ -1,61 +1,50 @@
...
+    <FullHeightPanel
+      className={styles.encrypt}
+      headerText={intl.formatMessage({ id: 'encryptPanelHeader' })}
+      renderCloseButton={() => (
+        <CloseButton onClick={resetEncryptedWIF} routeTo={ROUTES.SETTINGS} />
+      )}
+      renderHeaderIcon={HeaderIcon}
+      renderInstructions={
+        encryptedWIF
+          ? () => intl.formatMessage({ id: 'encryptInstructions' })
```
Comment (comountainclimber) [inline app/containers/Encrypt/Encrypt.jsx:37]: We should skip the instructions part OR make the instructions say something different than "Choose a passphrase to encrypt an existing key" like "please save your information in a safe place" once they have successfully encrypted a key (at this point they have already chosen a passphrase)

Screen Shot 2022-09-01 at 10 28 35 AM

**Comment by comountainclimber** [comment]
Two minor things I noticed when reviewing this... Nice work @raulduartep !! 💪

File: app/hocs/withCopyCanvasToClipboard.js (hunk 1 above)
Comment (raulduartep) [reply app/hocs/withCopyCanvasToClipboard.js]: All right. I was a bit confused on how to get the storage path. Thanks for the alert, I will fix this.

File: app/containers/Encrypt/Encrypt.jsx (hunk 2 above)
Comment (raulduartep) [reply app/containers/Encrypt/Encrypt.jsx:37]: Ok. You are right. I will fix.

**Comment by comountainclimber** [comment]
@raulduartep please let me know when the feedback has been addressed and merge conflicts are resolved 👀

**Review by comountainclimber — APPROVED** [review]

**Pull request merged**